Model/Resolver/Shipments.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace ScandiPWA\SalesGraphQl\Model\Resolver;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Sales\Api\Data\ShipmentInterface;
+use Magento\Sales\Model\Order;
+use Magento\SalesGraphQl\Model\Resolver\Shipments as BaseShipments;
+
+/**
+ * Resolve shipment information for order
+ */
+class Shipments extends BaseShipments
+{
+    /**
+     * @inheritDoc
+     */
+    public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null)
+    {
+        if (!isset($value['model']) && !($value['model'] instanceof Order)) {
```

Comment:
probably want to check OrderInterface rather than instance of Order model (in case it is preferenced in a way that isn't extending base model, for example. unlikely but can happen)